package.json

```diff
@@ -106,6 +106,17 @@
                                 "type": "number",
                                 "description": "A port to be used to enable automatic reloading of breakpoints when sourcemaps change.",
                                 "default": 9090
+                            },
+                            "sourceMaps": {
+                                "type": "boolean",
+                                "default": true
```

Comment:
The "default" parameter should refer to what the code does if you do not specify the parameter, and thus the default for sourceMaps is actually false.